### PR TITLE
Wal 249

### PR DESCRIPTION
**Pull request by mseaward**
GPOS Wizard landing page.
[WAL-249](https://peerplays.atlassian.net/browse/WAL-249)

### CODE REVIEW

File: src/components/Modal/GPOSWizard/GPOSWizard.scss [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+    justify-content: center;
+    align-items: center;
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: 2;
+    margin: 0 auto;
+    background: rgba(0, 0, 0, 0.5);
```
Comment (aametzler) [inline src/components/Modal/GPOSWizard/GPOSWizard.scss]: should be in constant file?

File: src/components/Modal/GPOSWizard/GPOSWizard.scss [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+    @include gpos-start-btn();
+
+    &-cancel, &-up, &-down {
+      @extend .gpos-modal__btn;
+      @include transition(all .2s ease);
+
+      border: 2px solid transparent;
+
+      &:active {
+        border-color: rgb(51, 183, 239);
```
Comment (aametzler) [inline src/components/Modal/GPOSWizard/GPOSWizard.scss]: throw color in constant file

File: src/components/Modal/GPOSWizard/GPOSWizard.scss [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+      }
+    }
+
+    &-cancel {
+      background: rgb(41, 58, 143);
```
Comment (aametzler) [inline src/components/Modal/GPOSWizard/GPOSWizard.scss]: throw color in constant file

File: src/components/Modal/GPOSWizard/GPOSWizard.scss [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+      }
+
+      &:active {
+        background: darken(rgb(41, 58, 143), $amount: 10%);
+        border-color: $color-add2-bd-color;
+      }
+    }
+
+    &-up {
+      background: rgb(51, 183, 239);
```
Comment (aametzler) [inline src/components/Modal/GPOSWizard/GPOSWizard.scss]: throw color in constant file

File: src/components/Modal/GPOSWizard/GPOSWizard.scss [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+
+      &:hover {
+        background: lighten(rgb(51, 183, 239), $amount: 10%);
+      }
+
+      &:active {
+        background: darken(rgb(51, 183, 239), $amount: 10%);
```
Comment (aametzler) [inline src/components/Modal/GPOSWizard/GPOSWizard.scss]: throw color in constant file

File: src/components/Modal/GPOSWizard/GPOSWizard.scss [hunk 6]
```diff
@@ -0,0 +1,316 @@
...
+    }
+
+    &-down {
+      @include respond-to(mac) {
+        width: 172px;
+      }
+
+      width: 207px;
+      margin-right: 0;
+      background: rgb(19, 212, 183);
```
Comment (aametzler) [inline src/components/Modal/GPOSWizard/GPOSWizard.scss]: not going to comment every line. But just look at all the rgb references and throw them in a color constants file

**Comment by aametzler** [comment]
GPOS Wizard Modal should close when the session expires. Just had my session expire and the modal didn't close :)

**Comment by mseaward** [comment]
recent changes make this redundant, will have new branch to reflect the new changes

**Pull request closed without merging**